letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -530,7 +542,19 @@ CheckPathPermissions() {
 UNLIKELY_EOF
 }
 
-if [ "$1" = "--le-auto-phase2" ]; then
+if [ "$1" = "--le-auto-phase2" -a "$DEPRECATED_OS" = 1 ]; then
```

Comment:
nit: What do you think about moving this as a case of the existing phase 2 logic? I personally find it a little simpler to understand than adding a 3rd high level case and it deduplicates some code.

If we do this, at the end of the `DEPRECATED_OS = 1` case, we'll need to call `exit`.

Reply:
Done.